docs/tests/integration/test_wazuh_db/test_wazuh_db.md

```diff
@@ -0,0 +1,61 @@
+# Test Wazuh DB
+
+All these tests are meant to send a specific command to the socket (valid or not) and compare the resulting output with the expected result.
+## Overview
+
+Lets consider an example:
+```
+  name: "Update commands"
+  description: "Check success use cases for update commands on global DB"
+  test_case:
+  -
+    input: 'global update-agent-name {"id":1,"name":"TestName2"}'
+    output: "ok"
+    stage: "global update-agent-name success"
+```
+
+All similar test are grouped by a name, like `Update commands`. Then, every test has:
+
+- **input**: the command that will be sent to the socket
+- **output**: the expected result
+- **stage**: the name of that particular test
+## Objective
+
+Confirm that `wazuh-db` is able to save, update and erase the necessary information into the corresponding databases, using the proper commands and response strings.
+
+## General info
+
+|Tier | Number of tests | Time spent |
+|:--:|:--:|:--:|
+| 0 | 30 | 45s |
+
+## Expected behavior
+
+- Fail if `wazuh-db` response is different from output
+## Testing
+
+Tests description according to its classification
+### Checks agent_messages
+
+Currently, only **insert** and **clear** commands for `vuln_cves` table are tested
```

Comment:
I think if we avoid 
_Currently, only_ part will make this documentation easier to update.
i.e.:  _**insert** and **clear** commands for `vuln_cves` table are tested._

Reply:
You are right, fixed!